packages/components/drawer/src/DrawerContent.tsx

```diff
@@ -0,0 +1,28 @@
+import * as RadixDrawer from '@radix-ui/react-dialog'
+import { forwardRef, type ReactElement, type Ref } from 'react'
+
+import { drawerContentStyles, type DrawerContentStylesProps } from './DrawerContent.styles'
+
+export type ContentProps = RadixDrawer.DialogContentProps & DrawerContentStylesProps
+
+export const Content = forwardRef(
+  (
+    { children, className, size = 'md', side = 'right', ...rest }: ContentProps,
+    ref: Ref<HTMLDivElement>
+  ): ReactElement => (
```

Comment:
`ReactElement` not needed here

Reply:
It is all copy/paste from the dialog I will do a PR to remove them in both components.